html/modules/custom/reliefweb_topics/src/Plugin/Filter/IFrameFilter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\reliefweb_topics\Plugin\Filter;
+
+use Drupal\filter\FilterProcessResult;
+use Drupal\filter\Plugin\FilterBase;
+
+/**
+ * Provides a filter to allow iframes.
+ *
+ * @Filter(
+ *   id = "filter_iframe",
+ *   title = @Translation("IFrame Filter"),
+ *   description = @Translation("Process iframes"),
+ *   type = Drupal\filter\Plugin\FilterInterface::TYPE_MARKUP_LANGUAGE,
+ * )
+ */
+class IFrameFilter extends FilterBase {
```

Comment:
This `token_markdown` text format can be used elsewhere (ex: book pages, guidelines when introduced etc.) so I would suggest to move it this IframeFilter to the `reliefweb_utility` module.